backend/effects/builtin/toggle-command.js

```diff
@@ -0,0 +1,112 @@
+        name: "Toggle Command",
+        description: "Toggle a command's active status",
+        icon: "fad fa-toggle-off",
+        categories: [EffectCategory.COMMON],
+        dependencies: [],
+        triggers: effectModels.buildEffectTriggersObject(
+            [ControlKind.BUTTON, ControlKind.TEXTBOX],
+            [InputEvent.MOUSEDOWN, InputEvent.KEYDOWN, InputEvent.SUBMIT],
+            EffectTrigger.ALL
+        )
+    },
+    globalSettings: {},
+    optionsTemplate: `
+        <eos-container>
+            <p>This effect lets you automatically toggle the active status of Commands.</p>
+        </eos-container>
+
+        <eos-container header="Command Type" pad-top="true">
+            <dropdown-select options="{ system: 'System', custom: 'Custom'}" selected="effect.commandType"></dropdown-select>
+        </eos-container>
+
+        <eos-container ng-show="effect.commandType === 'system'" header="System Commands" pad-top="true">
+            <ui-select ng-model="effect.commandId" theme="bootstrap" ng-click="updateSelectedCommand(systemCommands)">
+                <ui-select-match placeholder="Select or search for a command... ">{{$select.selected.trigger}}</ui-select-match>
+                <ui-select-choices repeat="command.id as command in systemCommands | filter: { trigger: $select.search }" style="position:relative;">
+                    <div ng-bind-html="command.trigger | highlight: $select.search"></div>
+                </ui-select-choices>
+            </ui-select>
+        </eos-container>
+
+        <eos-container ng-show="effect.commandType === 'custom'" header="Custom Commands" pad-top="true">
+            <ui-select ng-model="effect.commandId" theme="bootstrap" ng-click="updateSelectedCommand(customCommands)">
+                <ui-select-match placeholder="Select or search for a command... ">{{$select.selected.trigger}}</ui-select-match>
+                <ui-select-choices repeat="command.id as command in customCommands | filter: { trigger: $select.search }" style="position:relative;">
+                    <div ng-bind-html="command.trigger | highlight: $select.search"></div>
+                </ui-select-choices>
+            </ui-select>
+        </eos-container>
+
+        <eos-container header="Toggle Action" pad-top="true">
+            <dropdown-select options="toggleOptions" selected="effect.toggleType"></dropdown-select>
+        </eos-container>
+    `,
+    optionsController: ($scope, commandsService) => {
+        $scope.systemCommands = commandsService.getSystemCommands();
+        $scope.customCommands = commandsService.getCustomCommands();
+
+        $scope.updateSelectedCommand = commands => {
+            if ($scope.effect.commandId) {
+                for (const command of commands) {
+                    if (command.id === $scope.effect.commandId) {
+                        $scope.effect.command = command;
+                    }
+                }
+            }
+        };
```

Comment:
We'll want to avoid storing the entire command object inside the effect data as it will eventually lead to stale data that can cause unexpected behavior.

Example of potential issue: User creates a Toggle Command effect, then later they edit the command to change its cooldown. Now when that Toggle Command effect runs, it will override all data for that command including replacing the cooldown with the old value.

Instead we should be able to rely on the already present `effect.commandId` and `effect.commandType` to grab current command data at effect run time. I detail this below.